packages/select/src/FilterMultiSelect/provider/SelectionProvider/SelectionProvider.tsx

```diff
@@ -12,7 +12,10 @@ export interface SelectionProviderProps {
   children: React.ReactNode // control how menu should look like
   items?: ItemType[]
   onSelectionChange?: (keys: Selection) => void
+  /** The currently selected keys in the collection (controlled). */
   selectedKeys?: Selection
+  /** The initial selected keys in the collection (uncontrolled). */
```

Comment:
Ooo I dunno, none of our other components support being uncontrolled 🤔

Reply:
Yeah I was wondering this - Is there a reason that its not recommended??

Reply:
Technically our Select and filterSelect allow `defaultSelectedKey` because we are extending the props :P

Reply:
I think @HeartSquared mentioned that no one wanted to use our components uncontrolled, so it never got built? 🤷‍♂️

In any case, seems we inadvertently support it with React Aria components, so why not. Filters = uncontrolled lol